docs/apireference/build/node_modules/jquery.scrollto/CHANGELOG.md

```diff
@@ -29,7 +29,7 @@
 ### Compat
 - Dropped support for $.scrollTo.window() and $(window)._scrollable()
 ### Fix
-- Now works consistenly on Chrome 40
+- Now works consistently on Chrome 40

```

Comment:
Can we add directories to exclude? This is a library, I believe

Reply:
nothing native in the command line .. but we are just piping the output of [find and excluding with sed](https://github.com/kubernetes/kops/pull/5277/files#diff-b67911656ef5d18c4ae36cb6741b7965R507) .. I'll add node_modules and revert the change